src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -346,11 +368,11 @@ export class OperationDetails {
         return result;
     }
 
-    private parseExamples(operation: Operation): void {
+    private parseResponseExamples(operation: Operation): void {
         const examples = operation.getMeaningfulResponses().reduce((acc, cur) => {
             const representations = cur.meaningfulRepresentations();
             if (!representations || !representations.length) return acc;
-
+            
```

Comment:
It is minor. Style.

Reply:
Let me merge that in such case and will be fixing in upcoming PR. on top of it, thank you!